### ISSUE TITLE
Add fields to comments for reporting

### ISSUE THREAD

**Issue by jaclynperrone** [hammerlab/cycledash#674]
Comments should work to improve the filtering/machine learning algorithm*. In order to do this, we will need some standardized data that a free-form text field can't give us. Therefore, we can incorporate an optional dropdown with some values (which are open to discussion!) to flag variants. Here are some examples:
[1] Real somatic
[2] Real germline
[3] Real loss of heterozygosity
[4] Artifact due to paralogous sequence misalignment
[5] Sequencing artifact
[6] Low complexity region
[7] Region that might benefit from re-assembly and re-calling
, in addition to a way to enter a confidence level.

The user can then use the free-form text field to explain their rationale behind their selections.

*I don't know how this will improve the filtering/machine learning algorithm (if it is even a thing that exists and/or can be improved right now). Would love to hear your thoughts on this.

**Comment by jaclynperrone** [comment]
Here's a possible design for this flow. When a user goes to make a new comment, there would be the two additional fields mentioned above **[1]**. Both fields are optional. Users can provide a classification without a confidence level, but they cannot provide a confidence level without a classification. (Perhaps the confidence level is disabled unless they select a classification.) The values **[2]** for these fields are placeholders and up for discussion/testing. The classifications came out of user interviews and I researched common values for confidence levels.

**[1]**
<img width="965" alt="1" src="https://cloud.githubusercontent.com/assets/5685044/10227466/74200130-683b-11e5-9c43-1e66c5b85ea8.png">

**[2]**
<img width="962" alt="2" src="https://cloud.githubusercontent.com/assets/5685044/10227514/c82f1b44-683b-11e5-942e-ff07b1f35175.png">

<img width="963" alt="3" src="https://cloud.githubusercontent.com/assets/5685044/10227523/cd59d212-683b-11e5-8a4c-7e79b4d765e1.png">

When viewing the comments on the Examine page, the user's selections would appear with a little less emphasis, placing the focus on the comment they've written. The mockup **[3]** shows the three possible comment views (with the 2 optional fields filled out, with 1 optional field, with 0 optional fields).

**[3]**
<img width="721" alt="4" src="https://cloud.githubusercontent.com/assets/5685044/10227602/4c5f3dd6-683c-11e5-92ea-3e5626d81cad.png">

And here is the new design applied to the 'All Comments' page **[4]**

**[4]**
<img width="1117" alt="5" src="https://cloud.githubusercontent.com/assets/5685044/10227845/b3adb732-683d-11e5-880c-d4285189f170.png">

Feedback is welcome!

**Comment by hammer** [comment]
Very interesting, thanks for mocking up @jaclynperrone! Do you think this variant classification system would complement starring variants or do you think stars don't have a purpose with this new system?

**Comment by jaclynperrone** [comment]
@hammer You're welcome!

I always thought of starring variants as an easy way to "bookmark" ones that you might not necessarily have a comment/opinion about. It's for variants and comment threads that you would like to come back to later. In this regard, this variant classification system would exist alongside a starring system.

At this point, the above is an assumption. The starring mechanism is something that would benefit from some testing. It's also something we can leave out until we get feedback that it's needed.

**Comment by hammer** [comment]
Cool just wanted to get your vision. In my head they're complementary, and I'd love to see us get starring in ASAP. Any chance we could push on the implementation? I know that @danvk and @ihodes have been busy with pileup and RGC respectively--perhaps we could entice them to knock starring out soon before pushing on this one?

**Comment by jaclynperrone** [comment]
Yeah! It would great to get starring done before this classification feature. There's a mockup for it in #675, and I moved it into the project milestone.
